src/main/ruby/core/posix.rb

```diff
@@ -18,6 +18,12 @@ module Truffle::POSIX
     LIBTRUFFLEPOSIX = LIBC
   end
 
+  if Rubinius.linux?
+    LIBCRYPT = Truffle::Interop.eval('application/x-native', 'load libcrypt.so')
```

Comment:
Is `load` an NFI command? I've never seen that before.

Reply:
Yes, and it's the same for LIBTRUFFLEPOSIX above.
The NFI language only understands `load LIBRARY` and `default` currently, fairly simple.